module/gnb/api/procedure.go

```diff
@@ -335,6 +335,172 @@ func DeRegistration(ue *RanUeContext) error {
 
 }
 
+// TestGUTIRegistration implements a function registartion using the GUTI after using SUPI reg
+func TestGUTIRegistration() error {
```

Comment:
Do not implement test function please, implement procedures such as 

```golang
func GUTIRegistration(ue) error {
....
}
```

We will assume that the user would have already registered the UE and deregistered berfore